tudatBundle/pagmo2/include/pagmo/algorithms/giACOmo.hpp

```diff
@@ -79,73 +79,73 @@ class giACOmo
+    * @param[in] ker Kernel: number of solutions stored in the solution archive
+    * @param[in] oracle Oracle parameter: this is the oracle parameter used in the penalty method
+    * @param[in] paretomax Max number of non-dominated solutions: this regulates the max number of Pareto points to be stored
+    * @param[in] epsilon Pareto precision: the smaller this parameter, the higher the chances to introduce a new solution in the Pareto front
     * @param seed seed used by the internal random number generator (default is random)
-    * @throws std::invalid_argument if \p acc is not \f$ \in [0,1[\f$, \p FSTOP is not positive, \p IMPSTOP is not a
-    * positive integer, \p EVALSTOP is not a positive integer, \p FOCUS is not \f$ \in [0,1[\f$, \p ANTS is not a positive integer,
-    * \p KER is not a positive integer, \p ORACLE is not positive, \p PARETOMAX is not a positive integer, \p EPSILON is not \f$ \in [0,1[\f$
+    * @throws std::invalid_argument if \p acc is not \f$ \in [0,1[\f$, \p fstop is not positive, \p impstop is not a
+    * positive integer, \p evalstop is not a positive integer, \p focus is not \f$ \in [0,1[\f$, \p ants is not a positive integer,
+    * \p ker is not a positive integer, \p oracle is not positive, \p paretomax is not a positive integer, \p epsilon is not \f$ \in [0,1[\f$
     */
-    giACOmo(unsigned gen = 1u, double acc = 0.95, unsigned int FSTOP = 1, unsigned int IMPSTOP = 1, unsigned int EVALSTOP = 1,
-          double FOCUS = 0.9,  unsigned int KER = 10, double ORACLE=1.0, unsigned int PARETOMAX = 10,
-            double EPSILON = 0.9, unsigned seed = pagmo::random_device::next())
-        : m_gen(gen), m_acc(acc), m_FSTOP(FSTOP), m_IMPSTOP(IMPSTOP), m_EVALSTOP(EVALSTOP), m_FOCUS(FOCUS),
-          m_KER(KER), m_ORACLE(ORACLE), m_PARETOMAX(PARETOMAX), m_EPSILON(EPSILON), m_e(seed), m_seed(seed), m_verbosity(0u),
+    gi_aco_mo(unsigned gen = 1u, double acc = 0.95, unsigned int fstop = 1, unsigned int impstop = 1, unsigned int evalstop = 1,
+          double focus = 0.9,  unsigned int ker = 10, double oracle=1.0, unsigned int paretomax = 10,
+            double epsilon = 0.9, unsigned seed = pagmo::random_device::next())
+        : m_gen(gen), m_acc(acc), m_fstop(fstop), m_impstop(impstop), m_evalstop(evalstop), m_focus(focus),
+          m_ker(ker), m_oracle(oracle), m_paretomax(paretomax), m_epsilon(epsilon), m_e(seed), m_seed(seed), m_verbosity(0u),
           m_log()
     {
         if (acc >= 1. || acc < 0.) {
             pagmo_throw(std::invalid_argument, "The accuracy parameter must be in the [0,1[ range, while a value of "
                                                    + std::to_string(acc) + " was detected");
         }
-        if (FSTOP < 0.) {
+        if (fstop < 0.) {
             pagmo_throw(std::invalid_argument, "The objective stopping criterion must be in the [0,inf[ range, while a value of "
```

Comment:
[0, inf[ is ok in math, for code please use >=0.
also fstop is unsigned so the check will always be true. 

I think this check can be removed.